Extension/test/scenarios/SingleRootProject/tests/extension.test.ts

```diff
@@ -115,7 +116,7 @@ suite("Pick Process Tests", () => {
 
         const process1: Process = parsedOutput[0];
         const process2: Process = parsedOutput[1];
-        const process3: Process = parsedOutput[2];
+        // const process3: Process = parsedOutput[2];
```

Comment:
Should this be removed or can a comment be added explaining when it should be uncommented?

Reply:
removed